nbbuild/build.xml

```diff
@@ -460,6 +465,8 @@ Hg ID:    ${hg.id}
           <property name="cluster.config" value="platform"/>
           <property name="test.type" value="test-unit"/>
           <property name="test-unit-sys-prop.ignore.random.failures" value="true"/>
+          <property name="permit.jdk9.builds" value="true"/>
```

Comment:
For completeness, as I believe this is obvious, the purpose of permit.jdk9.builds is to avoid producing builds that are not guaranteed to run on JDK 8. (I.e. explicitly accepting this fact the build won't necessarily be suitable for JDK 8, to avoid surprises.)